SignupDirector/SignupDirectorAPI/migrations/0002_participants_name_alter_participants_accompanions_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.1.7 on 2023-02-18 09:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('SignupDirectorAPI', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='participants',
+            name='name',
+            field=models.CharField(default='no_name', max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='participants',
+            name='accompanions',
+            field=models.IntegerField(default=0),
```

Comment:
there's no much reason to add a field and the alter it in the same PR

Reply:
done